test/built-ins/Temporal/Instant/prototype/subtract/constrain-when-ambiguous-result.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.subtract
+description: Constrains when ambiguous result.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const mar31 = Temporal.ZonedDateTime.from("2020-03-31T15:00+00:00[UTC]");
+
+TemporalHelpers.assertZonedDateTimesEqual(
+    mar31.subtract({ months: 1 }),
+    Temporal.ZonedDateTime.from("2020-02-29T15:00:00+00:00[UTC]"));
+
+TemporalHelpers.assertZonedDateTimesEqual(
+    mar31.subtract({ months: 1 }, { overflow: "constrain" }),
+    Temporal.ZonedDateTime.from("2020-02-29T15:00:00+00:00[UTC]"));
```

Comment:
I don't care too much about this so feel free to leave it, but you could extract the ZonedDateTime into `expected` like in the corresponding `add/` test.

Reply:
Done